internal/plugin/command_list.go

```diff
@@ -16,6 +16,7 @@ import (
 
 type out struct {
 	PluginName string `human:"Plugin Name" serialized:"plugin_name"`
+	PluginId   string `human:"Plugin Id" serialized:"plugin_id"`
```

Comment:
```suggestion
	PluginId   string `human:"Plugin ID" serialized:"plugin_id"`
```

Reply:
I wonder if we should just say "Name" and "ID" here to save space. The command is named `confluent plugin list` so it's clear that what we're listing are plugins. @sgagniere would need to incorporate this into the v4 release though, since it's a breaking change.

Reply:
i have changed the name to ID and Name for now. I will revert back if we decide otherwise